src/styles/color.ts

```diff
@@ -0,0 +1,31 @@
+const colors = {
+  primary: '#638FFF',
+  primary_50: '#F2F5FF',
+  primary_100: '#D8E3FF',
+  primary_200: '#638FFF',
+  primary_300: '#4F7BEB',
+
+  secondary: '#1D2942',
+  secondary_100: '#E7E9EC',
+  secondary_200: '#1D2942',
+  secondary_300: '#111929',
+
+  gray_50: '#F4F5F9',
+  gray_100: '#EDF0F5',
+  gray_200: '#E4E7EE',
+  gray_300: '#C9CFDF',
+  gray_400: '#677189',
+  gray_500: '#394258',
+
+  black: '#17171B',
+  white: '#ffffff',
+
+  red: '#F85B81',
+  pink: '#F4E0F9',
+  purple: '#ECE7F7',
+  yellowgreen: '#EBF7DA',
+  bluegreen: '#DEF3EC',
+  skyblue: '#E3F1FF',
+};
```

Comment:
```suggestion
}; as const
```

readonly 처리하면 어떨까요?!